recipes/boost/all/dependencies/dependencies-1.90.0.yml

```diff
@@ -0,0 +1,292 @@
+configure_options:
+- atomic
+- charconv
+- chrono
+- cobalt
+- container
+- context
+- contract
+- coroutine
+- date_time
+- exception
+- fiber
+- filesystem
+- graph
+- graph_parallel
+- iostreams
+- json
+- locale
+- log
+- math
+- mpi
+- nowide
+- process
+- program_options
+- python
+- random
+- regex
+- serialization
+- stacktrace
+- test
+- thread
+- timer
+- type_erasure
+- url
+- wave
+dependencies:
+  atomic: []
+  charconv: []
+  chrono: []
+  cobalt:
+  - container
+  - context
+  cobalt_io:
+  - cobalt
+  cobalt_io_ssl:
+  - cobalt
```

Comment:
```suggestion
```

Same comment as done for 1.89.0: https://github.com/conan-io/conan-center-index/pull/28187#pullrequestreview-3154859499